operator/pkg/test_samples/helm/crds/sample_crd.yaml

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
this crd should not be removed. Otherwise the sample Manifest CR will not work out of the box.